sdk/typescript/test/e2e/event-subscription.test.ts

```diff
@@ -1,45 +1,46 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { describe, it, expect, beforeAll, vi } from 'vitest';
-import { SuiEvent, Transaction } from '../../src';
-import { setup, TestToolbox } from './utils/setup';
+import { describe, beforeAll } from 'vitest';
+// import { SuiEvent, Transaction } from '../../src';
+// import { setup, TestToolbox } from './utils/setup';
 
 describe('Event Subscription API', () => {
-  let toolbox: TestToolbox;
+  // let toolbox: TestToolbox;
 
   beforeAll(async () => {
-    toolbox = await setup();
+    // toolbox = await setup();
   });
 
-  const mockCallback = vi.fn((_: SuiEvent) => expect(true).toBeTruthy());
+  // TODO: rewrite this test after the removal of devnet_nft
+  // const mockCallback = vi.fn((_: SuiEvent) => expect(true).toBeTruthy());
 
-  it('Subscribe to events', async () => {
-    const subscriptionId = await toolbox.provider.subscribeEvent({
-      filter: { Sender: toolbox.address() },
-      onMessage: mockCallback,
-    });
+  // it('Subscribe to events', async () => {
+  //   const subscriptionId = await toolbox.provider.subscribeEvent({
+  //     filter: { Sender: toolbox.address() },
+  //     onMessage: mockCallback,
+  //   });
 
-    const tx = new Transaction();
-    tx.moveCall({
-      target: '0x2::devnet_nft::mint',
-      arguments: [
-        tx.pure('Example NFT'),
-        tx.pure('An NFT created by the wallet Command Line Tool'),
-        tx.pure(
-          'ipfs://bafkreibngqhl3gaa7daob4i2vccziay2jjlp435cf66vhono7nrvww53ty',
-        ),
-      ],
-    });
-    await toolbox.signer.signAndExecuteTransaction({
-      transaction: tx,
-      requestType: 'WaitForLocalExecution',
-    });
+  //   const tx = new Transaction();
+  //   tx.moveCall({
+  //     target: '0x2::devnet_nft::mint',
+  //     arguments: [
+  //       tx.pure('Example NFT'),
+  //       tx.pure('An NFT created by the wallet Command Line Tool'),
+  //       tx.pure(
+  //         'ipfs://bafkreibngqhl3gaa7daob4i2vccziay2jjlp435cf66vhono7nrvww53ty',
+  //       ),
+  //     ],
+  //   });
+  //   await toolbox.signer.signAndExecuteTransaction({
+  //     transaction: tx,
+  //     requestType: 'WaitForLocalExecution',
+  //   });
 
-    const subFoundAndRemoved = await toolbox.provider.unsubscribeEvent({
-      id: subscriptionId,
-    });
-    expect(subFoundAndRemoved).toBeTruthy();
-    expect(mockCallback).toHaveBeenCalled();
-  });
+  //   const subFoundAndRemoved = await toolbox.provider.unsubscribeEvent({
```

Comment:
Looks like with this change we don't have any websocket tests? I'd probably find a way to at least add one websocket test so that we don't entirely regress that behavior.

Reply:
Yeah I will move the devnet-nft.move to examples folder and we can just publish that package in `beforeAll`. I'll add it back by the end of day